### PR TITLE
Missions

### PR DESCRIPTION
**Pull request by SamanAtashi**
# React & Redux group project - Space Travelers' Hub:
> Features introduced are as follows:
1. Use React documentation.
2. Use React components.
3. Use React props.
4. Use React Router.
5. Connect React and Redux.
6. Handle events in a React app.
7. Write unit tests with React Testing Library.
8. Use styles in a React app.
9. Use React life cycle methods.
10. Apply React best practices and language style guides in code.
11. Use store, actions and reducers in React.
12. Perform a code review for a team member.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json [hunk 1]
```diff
@@ -3,25 +3,32 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@craco/craco": "^6.4.0",
+    "@reduxjs/toolkit": "^1.6.2",
```
Comment (mikenath223) [inline package.json:7]: - [ ] Kindly update the readme following [this template](https://github.com/microverseinc/readme-template).

**Review by mikenath223 — CHANGES_REQUESTED** [review]
.

File: src/Components/Missions/Missions.js [hunk 2]
```diff
@@ -1,9 +1,88 @@
+/* eslint-disable no-tabs */
+/* eslint-disable no-mixed-spaces-and-tabs */
+/* eslint-disable react/button-has-type */
 import React from 'react';
+import { useSelector } from 'react-redux';
+// import Table from 'react-bootstrap/Table';
+// import { Badge, Button } from 'react-bootstrap';
```
Comment (mikenath223) [inline src/Components/Missions/Missions.js]: - [ ] Please remove this and all other commented code. 
- [ ] Also remove the eslint disable line, its best to fix the issue in the code.

File: package.json (hunk 1 above)
Comment (Lameck1) [reply package.json:7]: The README is now professionally documented. ✔️

File: src/Components/Missions/Missions.js (hunk 2 above)
Comment (Lameck1) [reply src/Components/Missions/Missions.js]: This one is also properly addressed ✔️

**Review by Lameck1 — APPROVED** [review]
Hi, @SamanAtashi & @TSHEPO-CLOUD,

You've done well addressing concerns in the previous review 👏🏼.
Your project is now complete!  Consider merging this branch :shipit: 

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**